src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs

```diff
@@ -66,124 +66,83 @@ public async Task SetAfterUse_Throws()
             server => server.AcceptConnectionSendResponseAndCloseAsync());
         }
 
-        [OuterLoop]
-        [Fact]
-        public async Task InfiniteSingleHeader_ThrowsException()
+        [Theory]
+        [InlineData(1)]
+        [InlineData(15)]
+        public async Task LargeSingleHeader_ThrowsException(int? maxResponseHeadersLength)
```

Comment:
All the test parameters have a value. Does this need to be nullable?

Reply:
Not anymore (I was initially testing with the default value as well)